Client/Frontend/AuthenticationManager/SensitiveViewController.swift

```diff
@@ -17,7 +17,7 @@ class SensitiveViewController: UIViewController {
 
         willEnterForegroundNotificationObserver = NotificationCenter.default.addObserver(forName: UIApplication.willEnterForegroundNotification, object: nil, queue: .main) { [self] notification in
             if !isAuthenticated {
-                AppAuthenticator.authenticateWithDeviceOwnerAuthentication { [self] result in
+                AppAuthenticator().authenticateWithDeviceOwnerAuthentication { [self] result in
```

Comment:
```suggestion
                AppAuthenticator().authenticateWithDeviceOwnerAuthentication { [weak self] result in
```

```suggestion
                AppAuthenticator().authenticateWithDeviceOwnerAuthentication { [unowned self] result in
```

nit: good to have self captured

Reply:
You can pick the one depending on your needs if self is nil or can become nil then go with weak